lookout/style/format/analyzer.py

```diff
@@ -344,6 +365,40 @@ def _generate_token_fixes(
             ))
         return token_fixes, new_vnodes, y_pred_pure, y
 
+    @staticmethod
+    def get_train_test_split(files: Sequence[File], test_dataset_ratio: float,
+                             min_files_number: int,
+                             random_state: int) -> Tuple[Sequence[File], Sequence[File]]:
+        """
+        Create train test split for the files collection.
+
+        File size is estimated by its length.
+
+        :param files: The list of `File`-s (see service_data.proto) of the same language.
+        :param test_dataset_ratio: The fraction of data that should be taken for test dataset.
+        :param min_files_number: The minimum number of files in the test dataset. \
+                                              Usually is used to get at least one file for repos \
+                                              with small number of files.
+        :param random_state: Random state.
+        :return: Train files and test files.
+        """
+        min_files_number = min(min_files_number, len(files) - 1)
+        if test_dataset_ratio == 0:
+            return files, []
+        random.seed(random_state)
+        files = random.sample(files, k=len(files))
+        target_train_length = sum(map(lambda f: len(f.content), files)) * (1 - test_dataset_ratio)
+
+        accumulated_train_length = 0
+        i = 0
```

Comment:
`i = 0` is not needed, variables are defined along the way in python

Reply:
if a list is empty `i` will not be defined.